modules/base/rendering/pointcloud/renderablepointcloud.h

```diff
@@ -118,10 +119,17 @@ class RenderablePointCloud : public Renderable {
     /// Find the index of the currently chosen size parameter in the dataset
     int currentSizeParameterIndex() const;
 
+    bool hasColorData() const;
```

Comment:
Could these functions be made `private` or at least `protected`?

Reply:
They are protected :)

Reply:
🤦